bin/hdfcoinc/pycbc_multiifo_coinc_findtrigs

```diff
@@ -234,7 +240,7 @@ for tnum in template_ids:
         logging.info('%s:%s' % (ifo, len(times_full[ifo])))
 
     logging.info('Calculating Single Detector Statistic')
-    sds_full = {ifo:rank_method.single(trigs[ifo]) for ifo in trigs}
+    sds_full = {ifo: rank_method.single(trigs[ifo]) for ifo in trigs}
```

Comment:
Shouldn't be a space after 'ifo:'

Reply:
Going by 
https://stackoverflow.com/questions/9253432/coding-guidelines-for-python-dictionaries